brave-spark-framework-filter/src/test/java/com/github/kristofa/brave/spark/ITBraveSparkExample.java

```diff
@@ -0,0 +1,74 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.LoggingReporter;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import spark.Spark;
+import zipkin.reporter.Reporter;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by 00013708 on 2017/3/13.
+ */
+public class ITBraveSparkExample {

```

Comment:
in order to test all of the behaviors needed, you'd subclass ITHttpServer similar to how others are doing.